File: Python/1684_count_the_number_of_consistent_strings.py
class Solution:
    def countConsistentStrings(self, allowed: str, words: list[str]) -> int:
        count = 0
        valid = 1
        for word in words:
            for char in word:
                if char not in allowed:
                    valid = 0
            if valid:
                count += 1
            valid = 1
        return count

File: JavaScript/1684_count_the_number_of_consistent_strings.js
/**
 * @param {string} allowed
 * @param {string[]} words
 * @return {number}
 */
var countConsistentStrings = function(allowed, words) {
    let consistentWords = 0;
    for (const word of words) {
        let valid = true;
        for (char of word) {
            if (!allowed.includes(char)) {
                valid = false;
            }
        }

        if (valid) {
            consistentWords += 1;
        }
    }
    return consistentWords;
};

